feature-group-definitions/background-clip.yml

```
@@ -0,0 +1,8 @@
+name: background-clip
+description: The `background-clip` CSS property allows the background to be clipped to the padding box or content box, in addition to the default of the whole border box.
```

Like above, what do you think of this, to dodge the "allows" bit. (A variation, to see an alternative to "drawing" here.)

```suggestion
description: The `background-clip` CSS property sets the extent of the background: the padding box, the content box, or the default border box.
```

---

Thanks, I like this rewrite!